ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -342,6 +342,9 @@ def test_sum(
     if "torch" in backend_fw:
         assume(not test_flags.as_variable[0])
         assume(not test_flags.test_gradients)
+    if "jax" in backend_fw and castable_dtype in ["complex32", "complex64"]:
```

Comment:
I think this should be `"complex32", "complex128"` instead, what do you think?

Reply:
> I think this should be `"complex32", "complex128"` instead, what do you think?

Ivy support `complex64` and `complex128` so maybe we should cater to that? Also the paddle backend gave this error so changed its decorator `ivy.utils.exceptions.IvyBackendException: paddle: execute_with_gradients: paddle: nested_map: paddle: mean: (NotFound) The kernel with key (CPU, Undefined(AnyLayout), int32) of kernel `mean` is not registered. Selected wrong DataType `int32`. Paddle support following DataTypes: float64, complex128, float32, complex64, bool`
I dont know why `mean` is getting triggered even when we are dealing with `sum` function.